src/admin/components/ResourceTabs/ChangeRequestsTab/useFormChanges.ts

```diff
@@ -51,7 +50,7 @@ export default function useFormChanges(current: any, changed: any, steps: FormSt
               title: entry.title,
               type: entry.type,
               currentValue,
-              newValue: isEqual(currentValue, newValue) ? undefined : newValue
+              newValue: currentValue === newValue ? undefined : newValue
```

Comment:
Commented about this on the other PR: please revert this change, it will break existing functionality.

Reply:
done